### DISCUSSION TITLE
A safe way to ensure a module is always mocked

### DISCUSSION THREAD

**Discussion by revosw** [vitest-dev/vitest#5682] [Q&A] [answered]
I use Twilio to send SMS. Under no circumstance do I _not_ want to mock this module.

I had a recent mishap where an API endpoint was initially not sending SMS, but then I added SMS sending. I forgot to `vi.mock("twilio")` in the test file before running `npx vitest`.

What is the best way to ensure that no matter what, a module is always mocked?

**Comment by sheremet-va** [comment, marked as answer]
You can add `vi.mock('twilio')` to `setupFiles`
